src/v1/creative-auction-bulk/types.ts

```diff
@@ -40,12 +47,19 @@ export type CreativeAuctionBulkParams = {
 
 export type CreativeAuctionBulkHttpRequestBody = {
   request_id: string;
+  channel_type?:
+    | 'UNKNOWN_CHANNEL_TYPE'
```

Comment:
We don't need to expose `UNKNOWN_CHANNEL_TYPE`. If the channel_type is omitted from the request, it will be handled as `UNKNOWN_CHANNEL_TYPE` on the server-side.
```suggestion
    
```

Reply:
Fixed